libs/sdk-ui-kit/src/Dialog/ExportDialogBase.tsx

```diff
@@ -37,6 +38,8 @@ export const ExportDialogBase = React.memo<IExportDialogBaseProps>(function Expo
     const [isFilterContextIncluded, setIsFilterContextIncluded] = usePropState(includeFilterContext);
     const [shouldMergeHeaders, setShouldMergeHeaders] = usePropState(mergeHeaders);
 
+    const mergeHeadersId = `${useId()}-mergeHeaders`;
```

Comment:
Leave it as is but looking at this line, it would be more readable (to me), if the method would accept parameters, for example `const mergeHeadersId = useId("mergeHeaders");` which would generate `mergeHeaders_5454fd54fs5f45s` (or the param could be suffix and not prefix). This way, it could however not be replacable by React 19 import later. Still, we could use it inside of it.

btw why adding the suffix? You want it to be more readable in the DOM viewer?

Reply:
As mentioned I wanted to be fully compatible with our helper to the react implementation. Prefix/suffix is there only to make relations in DOM tree more readable as number of IDs and their usage in describeBy/labelBy/controls is growing some human readable part of ID can help you to navigate quicker by eye

Reply:
added useIdPrefixed with adviced notation